.github/workflows/black.yml

```diff
@@ -2,9 +2,11 @@ name: black
 
 on:
   push:
-    branches: [master, ]
+    branches:
+      - '**'
```

Comment:
it looked like this 

`on: [push, pull_request]`

before the most recent update in august.  I'm not sure why it stopped working though - and I thought this maintained the most flexibility and style with what you were going for in the most recent change.  This is essentially regex - in case we want to exclude or include branches in the future.

Reply:
LGTM.